pkg/k8s/pipelineRun_test.go

```diff
@@ -467,6 +455,7 @@ func Test_pipelineRun_GetPipelineRepoServerURL_WrongURLs(t *testing.T) {
 	}
 }
 
+/*
```

Comment:
Here we commented a test --> we should revisit how to activate the test again (or decide that tests is non-sense now ...)

Reply:
Test is working again. The expected error is now provided by `Commit()` rather then by `UpdateStatus()` (and the error is wrapped).